### PR TITLE
Expose version_id on google_cloudfunctions_function

### PR DESCRIPTION
**Pull request by drcapulet**
This was already in the YAML but the lookup has custom code so it wasn't exposed - updated the type to be an integer since that's what the Go SDK returns, but happy to keep it a string.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
Expose version_id on google_cloudfunctions_function
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/cloudfunctions/resource_cloudfunctions_function.go [hunk 1]
```diff
@@ -490,6 +490,11 @@ func ResourceCloudFunctionsFunction() *schema.Resource {
 				Computed:    true,
 				Description: `Describes the current stage of a deployment.`,
 			},
+			"version_id": {
+				Type:        schema.TypeInt,
```
Comment (melinath) [inline mmv1/third_party/terraform/services/cloudfunctions/resource_cloudfunctions_function.go:494]: It looks like this is a `string (int64)` in the API, so it should be represented as a string in Terraform for consistency. TypeInt can't reliably represent int64 values.

**Review by melinath — CHANGES_REQUESTED** [review]

File: mmv1/third_party/terraform/services/cloudfunctions/resource_cloudfunctions_function.go (hunk 1 above)
Comment (drcapulet) [reply mmv1/third_party/terraform/services/cloudfunctions/resource_cloudfunctions_function.go]: Sure, updated!

**Review by melinath — APPROVED** [review]

**Pull request merged**